common/src/main/java/org/dash/wallet/common/ui/enter_amount/EnterAmountFragment.kt

```diff
@@ -90,6 +95,12 @@ class EnterAmountFragment: Fragment(R.layout.fragment_enter_amount) {
 
         binding.keyboardView.onKeyboardActionListener = keyboardActionListener
         binding.continueBtn.setOnClickListener {
+            if (!binding.amountView.showCurrencySelector) {
+                analyticsService.logEvent(
+                    AnalyticsConstants.Coinbase.CONTINUE_DASH_PURCHASE,
```

Comment:
@ClaudeHangui this is still wrong. Common components should only be responsible for their own functionality and shouldn't know or care about event logging. Events should be propagated down to the end-user (conbase module in this case) and logged in there. 

Otherwise, we'll end up with a tree of if-else that checks for coinbase vs crowdnode vs etc logs. This breaks both single-responsibility and open-closed principles.

Reply:
This has been addressed, so I will merge this story.